### PR TITLE
updated winui3/reunion doc

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

**Review by david-risney — APPROVED** [review]

**Comment by jasonstephen15** [comment]
ready for doc team review @Reezaali

File: microsoft-edge/webview2/get-started/winui.md [hunk 1]
```diff
@@ -9,34 +9,14 @@ ms.prod: microsoft-edge
...
         
-## Step 0 - Visual Studio settings  
+## Step 0 - Set Up Development Environment 
 
-1.  Ensure your system has a NuGet package source enabled for [nuget.org][NugetHome].  For more information, navigate to [Common NuGet configurations][NugetConsumePackagesConfiguringNugetBehavior] and [Windows Community Toolkit][WindowsCommunitytoolkit].  
-1.  Download and install the [Project Reunion VSIX package][VisualstudioMarketplaceProjectreunionMicrosoftprojectreunion].  The installer adds both the WinUI 3 project templates and the NuGet package containing the WinUI 3 libraries to Visual Studio 2019.  
-    
-    For instructions on how to add the `VSIX` package to Visual Studio, navigate to [Finding and Using Visual Studio Extensions][VisualstudioIdeFindingUsingVisualStudioExtensionsInstallWithoutUsing-ManageExtensionsDialogBox].
-    
+1. Follow steps 1-4 of the [WindowsAppSDK environment setup guide][WindowsAppsWinui3ConfigureYourDevEnvironment]. This guide will walk you through minimum system requirements, installing visual studio, enabling NuGet package source, and installing the WindowsAppSDK Extension for Visual Studio.
```
Comment (vchapel) [inline microsoft-edge/webview2/get-started/winui.md:18]:
```suggestion
1. Follow steps 1-4 of the [WindowsAppSDK environment setup guide][WindowsAppsWinui3ConfigureYourDevEnvironment]. This guide describes the minimum system requirements, installing visual studio, enabling NuGet package source, and installing the WindowsAppSDK Extension for Visual Studio. 
```

File: microsoft-edge/webview2/get-started/winui.md [hunk 2]
```diff
@@ -9,34 +9,14 @@ ms.prod: microsoft-edge
...
-    
-    For instructions on how to add the `VSIX` package to Visual Studio, navigate to [Finding and Using Visual Studio Extensions][VisualstudioIdeFindingUsingVisualStudioExtensionsInstallWithoutUsing-ManageExtensionsDialogBox].
-    
+1. Follow steps 1-4 of the [WindowsAppSDK environment setup guide][WindowsAppsWinui3ConfigureYourDevEnvironment]. This guide will walk you through minimum system requirements, installing visual studio, enabling NuGet package source, and installing the WindowsAppSDK Extension for Visual Studio. 
+1. Install [WebView2 Runtime][Webview2Installer] or any [Microsoft Edge (Chromium) non-stable channel][MicrosoftedgeinsiderDownload] installed on Windows 10 version 1803 \(build 17134\) or later.  For more information about Windows 10, navigate to [Windows Update: FAQ][MicrosoftSupport12373].  
 1.  To access all developer-specific Visual Studio features, turn on [Developer Mode][WindowsUwpGetStartedEnableYourDeviceForDevelopment].  
     
 ## Step 1 - Create Project
```
Comment (vchapel) [inline microsoft-edge/webview2/get-started/winui.md:22]: Step 2 says "choose C#, Windows, and WinUI respectively" but the image (winui-getting-started-selections.png) seems to show C#, Windows, Desktop, and WinUI. Do the text or image need to be changed to match each other?

File: microsoft-edge/webview2/get-started/winui.md [hunk 3]
```diff
@@ -246,7 +233,7 @@ You may use host apps to inject JavaScript code into WebView2 controls at runtim
 As an example, add scripts that send an alert when a user navigates to non-HTTPS sites.  Modify the `EnsureHttps` function to inject a script into the web content that uses [ExecuteScriptAsync][Webviews2ReferenceWpfMicrosoftWebExecutescriptasync].
```
Comment (vchapel) [inline microsoft-edge/webview2/get-started/winui.md:233]: Also, in the second bullet under the "Scripting" heading, the second bullet: "Run it before any other script included in the HTML document is run." reads better as "Run it before any other script included in the HTML document runs."

File: microsoft-edge/webview2/get-started/winui.md (hunk 2 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/get-started/winui.md:22]: text can be changed to specify desktop

Comment (jasonstephen15) [reply microsoft-edge/webview2/get-started/winui.md:22]: Actually the drop down selections only have spots for C#, Windows, WinUI, there is no place to choose Desktop, In the highlighted selection that is just an extra tag. We can leave it as is for now.

File: microsoft-edge/webview2/get-started/winui.md (hunk 3 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/get-started/winui.md:233]: changed

**Comment by jasonstephen15** [comment]
Ready to merge @vchapel

**Comment by jasonstephen15** [comment]
Checking in here @Reezaali @vchapel any action needed on my end?

**Comment by vchapel** [comment]
> Checking in here @Reezaali @vchapel any action needed on my end?

Reeza is out this week and I listed him as a reviewer, so I think we just need to wait ... unless it is urgent.

**Pull request merged**